src/components/view/ListView.vue

```diff
@@ -229,6 +232,54 @@ export default {
     editValue (record) {
       this.editableValueKey = record.key
       this.editableValue = record.value
+    },
+    handleShowBadge () {
```

Comment:
This logic should ideally be moved to the action component (instead of bloating the list view component)